### ISSUE TITLE
Search Results – Should be able to use TAB key to toggle between the results

### ISSUE THREAD

**Issue by Layoric** [TerriaJS/terriajs#506]
When the user enters a search, the application displays a list. In the screen shot example below the user entered “Maranboy” as the search string.  User should be able to use the TAB key to move between the results, or even use the up and down arrow.

![issue-search-results](https://cloud.githubusercontent.com/assets/234642/5995621/701c4aba-aaec-11e4-87f3-d61979715e93.png)

**Comment by kring** [comment]
Thanks @Layoric, we definitely plan to do this.

**Comment by kring** [comment]
This was done in #966.
